### ISSUE TITLE
Template integration: feedback on article_list

### ISSUE THREAD

**Issue by ryanpitts** [OpenNews/opennews-source#70]
@beep @kissane article list with pagination (yay), cribbing the article module that appears at the bottom of a person's profile page for the pieces they've authored: https://opennews-source-staging.herokuapp.com/articles/

Wondering if we maybe want to work in pubdate for these, though. Perhaps in place of the subhead?

![screen shot 2017-02-01 at 2 25 30 pm](https://cloud.githubusercontent.com/assets/56477/22528963/5bf8bb4e-e88a-11e6-9c74-366d3fce8b51.png)

Also, on other landing pages we do have a "big" lead, like so:

![screen shot 2017-02-01 at 2 26 47 pm](https://cloud.githubusercontent.com/assets/56477/22529028/985c29d6-e88a-11e6-86c3-b9ed697b9c52.png)

But it feels like maybe we don't want that here? not sure.

**Comment by kissane** [comment]
Two **Priority 2** things I would suggest here, based on @ryanpitts' notes:

- @beep, may we add the big lead title-thing Ryan is suggesting upthread, à la Community, using "Articles" and some dumb text I will replace soon ("Projects walkthroughs, tool teardowns, interviews, and more")? And then suppress the smaller red Articles title/header?
- May we also replace the subhead with the pub date, as Ryan suggests above? "February 2, 2017" as the date style, I think?

**Comment by beep** [comment]
> @beep, may we add the big lead title-thing Ryan is suggesting upthread, à la Community, using "Articles" and some dumb text I will replace soon ("Projects walkthroughs, tool teardowns, interviews, and more")? And then suppress the smaller red Articles title/header?

@kissane Absolutely! Here’s a quick mockup I just did in-browser:

![screen shot 2017-02-02 at 12 21 09 pm](https://cloud.githubusercontent.com/assets/124493/22560318/271cff58-e942-11e6-91f8-d2601abc6e69.png)

@ryanpitts: I deleted the leading `.hed-label`, and dropped in a `h1.page-title-lead` and `div.mod.main-lead` where it stood. Let me know if a static template’d help.

> May we also replace the subhead with the pub date, as Ryan suggests above? "February 2, 2017" as the date style, I think?

Definitely! Any text can appear in that subhed block:

![screen shot 2017-02-02 at 12 22 56 pm](https://cloud.githubusercontent.com/assets/124493/22560392/6915eca8-e942-11e6-9cb0-abe427923a32.png)

Though I’d be happy to come up with a date-specific subhed style, if you’d like?

**Comment by kissane** [comment]
@beep, these look marvellous. I am, however, a moron. 

Could the date-line stay in exactly that same style, but also include the author name, which has gone missing somewhere between my brain and the page? So "By Mike Stucka, posted February 2, 2017"?

Facepalm.

**Comment by kissane** [comment]
And once this is sorted, I would love to echo the within-tile changes adding author and date to the Articles by Tag list, @ryanpitts, if we may. Not sure we need to pull in the new header/lead, though.

**Comment by beep** [comment]
> Could the date-line stay in exactly that same style, but also include the author name, which has gone missing somewhere between my brain and the page? So "By Mike Stucka, posted February 2, 2017"?

Yes! (Sorry, should’ve guessed you’d want that.) Here’s a sketch:

![screen shot 2017-02-02 at 12 32 17 pm](https://cloud.githubusercontent.com/assets/124493/22560723/a79e7a7a-e943-11e6-9496-0d690d1a8173.png)

![screen shot 2017-02-02 at 12 31 23 pm](https://cloud.githubusercontent.com/assets/124493/22560734/acc7c6aa-e943-11e6-845c-a1e75aca51e3.png)

So! Anything _can_ go in there. That said, it might get tricky when that block has more than a few authors in it, though?

![screen shot 2017-02-02 at 12 38 04 pm](https://cloud.githubusercontent.com/assets/124493/22560972/7c886520-e944-11e6-9f26-0a82ada5443b.png)

TLDR: @kissane, I wonder if we should do something like this?

![screen shot 2017-02-02 at 12 36 45 pm](https://cloud.githubusercontent.com/assets/124493/22560939/5bd067d8-e944-11e6-8cfd-dcb9bbc0fce2.png)

sorry for all the words, what do u think, y/n

**Comment by ryanpitts** [comment]
ooh, I dig the last one, with the pubdate as a gray leader into the summary graf

**Comment by kissane** [comment]
same same same same I love it yes please

**Comment by beep** [comment]
yay i will make now

**Comment by beep** [comment]
Added in c2795aca3724a851ffb911b1fbb54c5d27749abe.

**Comment by ryanpitts** [comment]
on staging now: https://opennews-source-staging.herokuapp.com/articles/

**Comment by kissane** [comment]
Looks awesome.
